exporters/dsexporter/dsexporter_test.go

```diff
@@ -2,45 +2,136 @@ package main
 
 import (
 	"testing"
+	"reflect"
+	"net/http"
+	"net/http/httptest"
 
-	"github.com/prometheus/client_golang/prometheus"
 	"github.com/stretchr/testify/assert"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/prometheus/client_golang/prometheus/testutil"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
 )
 
-func TestCustomCollector(t *testing.T) {
-	exporter := NewCustomCollector()
-	prometheus.MustRegister(exporter)
+func MockInclusterConfig() (*rest.Config) {
+    return nil
+}
+
+func MockNewForConfig(*rest.Config) (*kubernetes.Clientset) {
+	return nil
+}
+
+func TestNewKubeClient(t *testing.T) {
+	actualInclusterConfig := getInClusterConfig
```

Comment:
Fix indentation

Reply:
Maybe this line is actually with the correct indentation and the ones below are under-indented. Need to keep it consistent across the code.

Reply:
I think now that the problem is that you have a combination of tabs and spaces. You can configure your editor to automatically convert one to the other.
I think code bases that we're involved with are using spaces more often than tabs, so better use spaces all around and set the editor to convert tabs to spaces as you type them.